index.html

```diff
@@ -14,6 +16,100 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  
+  <!-- container -->
+  <div class="container"> 
+
+  <!-- header -->
+    <header>
+        <nav>
+            <a href="#">Home</a>        
+            <a href="#">About</a>
+            <a href="#">Portfolio</a>    
+            <a href="#">Contact</a>    
+        </nav>
+
+    </header>
+  <!-- header -->
+
+  <!-- Call to Action Section -->
```

Comment:
Loving all these comments explaining/labeling your blocks of code!!! great job on this